osu.Game/Overlays/OverlayColourProvider.cs

```diff
@@ -57,56 +55,22 @@ public void ChangeColourScheme(OverlayColourScheme colourScheme)
             ColourScheme = colourScheme;
         }
 
-        private Color4 getColour(float saturation, float lightness) => Color4.FromHsl(new Vector4(getBaseHue(ColourScheme), saturation, lightness, 1));
+        private Color4 getColour(float saturation, float lightness) => Framework.Graphics.Colour4.FromHSL(getBaseHue(ColourScheme), saturation, lightness);
```

Comment:
What's with the change in type? It's not a super big deal but this means it's implicitly converting from `Colour4` to `Color4`.

Reply:
https://discord.com/channels/188630481301012481/188630652340404224/1261708567711449179

Reply:
🙈